physics/CONV/Chikira_Sugiyama/cs_conv.F90

```diff
@@ -3841,64 +4083,66 @@ SUBROUTINE CUMFXR1                                   & ! Tracer mass fixer
         ! 2: mass fixer is applied, total mass never change through cumulus scheme
         !    e.g. CO2
       INTEGER      ISTS, IENS
+      REAL(kind_phys)     sigmai (IM,KMAX+1,NCTP), sigma(IM,KMAX+1)   !!DDsigma cloud updraft fraction
```

Comment:
Variables sigmai and sigma are not used in SUBROUTINE CUMFXR1, so variables sigmai, sigma and nctp can be removed from SUBROUTINE CUMFXR1